helix-core/src/test/java/org/apache/helix/model/TestInstanceConfig.java

```diff
@@ -90,6 +91,16 @@ public void testEnabledInstanceBackwardCompatibility() {
     Assert.assertFalse(instanceConfig.getInstanceEnabled());
   }
 
+  @Test
+  public void testInstanceOperationDeserialize() {
```

Comment:
Why we suddenly add this?

Reply:
Noticed we do not have test coverage for this..